build.gradle

```diff
@@ -47,7 +47,7 @@ buildscript {
   dependencies {
     classpath 'com.android.tools.build:gradle:2.3.0'
     classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
-    classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.8'
+    classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.10'
```

Comment:
Required for Gradle 3.5 compatibility: https://github.com/tbroyer/gradle-errorprone-plugin#requirements